### PR TITLE
feat(app): Updated modal components FRT-5778

### PR DESCRIPTION
**Pull request by Artem1211**

### CODE REVIEW

File: packages/app/src/features/liquidity-flow/components/opacity-css-transition/opacity-css-transition.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { TRANSITION_CLASS_NAME } from './constants'
+
+interface Props {
+  inTransition: boolean
+  children?: React.ReactNode
+}
+
+export function OpacityCssTransition({ inTransition, children }: Props) {
+  return (
+    <CSSTransition in={inTransition} timeout={300} classNames={TRANSITION_CLASS_NAME}>
```
Comment (bouvens) [inline packages/app/src/features/liquidity-flow/components/opacity-css-transition/opacity-css-transition.tsx:13]: Let's extract the magic number `300` to a constant.

File: packages/app/src/features/liquidity-flow/hooks/use-borrow-first-position.tsx [hunk 2]
```diff
@@ -91,7 +90,21 @@ export const useBorrowFirstPosition = (
...
+        open
+        maxWidth="md"
+      >
+        <OpacityCssTransition inTransition={step === Step.Deposit}>
+          {modalByStep[step]}
+        </OpacityCssTransition>
+        {/* <CSSTransition in={step === Step.Deposit} timeout={300} classNames="dialog" /> */}
+      </Modal>
+    )) ||
+    null
```
Comment (bouvens) [inline packages/app/src/features/liquidity-flow/hooks/use-borrow-first-position.tsx:107]:
```suggestion
    (step ? (
      <Modal
        onClose={close}
        onBack={step === Step.Deposit ? () => setStep(Step.Borrow) : undefined}
        open
        maxWidth="md"
      >
        <OpacityCssTransition inTransition={step === Step.Deposit}>
          {modalByStep[step]}
        </OpacityCssTransition>
        {/* <CSSTransition in={step === Step.Deposit} timeout={300} classNames="dialog" /> */}
      </Modal>
    )) :
    null
```

Comment (Artem1211) [reply packages/app/src/features/liquidity-flow/hooks/use-borrow-first-position.tsx]: Thx, fixed

File: packages/app/src/features/liquidity-flow/components/opacity-css-transition/opacity-css-transition.tsx (hunk 1 above)
Comment (Artem1211) [reply packages/app/src/features/liquidity-flow/components/opacity-css-transition/opacity-css-transition.tsx]: Added

**Review by bouvens — APPROVED** [review]

**Pull request merged**